modules/dynamic-operator/api/v1alpha1/kuberlogicservice_webhook.go

```diff
@@ -221,3 +238,15 @@ func validateScheduleFormat(schedule string) error {
 	_, err := cron.ParseStandard(schedule)
 	return err
 }
+
+func validateDomain(domain string) error {
+	klsList := &KuberLogicServiceList{}
+	options := client.MatchingLabels(map[string]string{"domain": domain})
```

Comment:
What if I edit domain spec field by hand and do not edit labels?

Reply:
If MatchingLabels needs only for filter services I do suggest do not use this labels, because it's duplicate the spec field.

Reply:
What about perfomance? In that case we will need to retrieve all of the
`kuberlogicservice` instances and check them one by one, because I didn't find a way to filter by spec fields

Reply:
We are far away from handred thousands services, but in case hundred services it's does not matter. Seems that premature optimization and I don't see how can you syncronize labels and specs - anyway, it's adds logic complication.

Reply:
Got it, will remove label matching

Reply:
Fixed